CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # astro changelog
 
-## 0.5.0 (UNRELEASED, 2018)
-
+## 0.5.0 (UNRELEASED, 2019)
+
+* Add `version` command
+* Propagate SIGINT and SIGTERM to terraform processes (#49)
+* Support detach flag with terraform 0.12 (#45)
+* Fix plan output for terraform 0.12 (#41)
+* Fix bug in initialization of allowed values (#43)
+* Don't pass varialbes to modules that don't delcare them (#40)
```

Comment:
Can we put this in a separate PR? They have nothing to do with this one.

(Also, typo: "variables")